.github/workflows/linux_sanitizers.yml

```diff
@@ -14,6 +27,14 @@ permissions: read-all
 
 env:
   TARGET_BRANCH: ${{ github.base_ref || github.event.merge_group.base_ref || github.ref }}
+  BUILD_DIR: /__w/openvino/openvino/openvino_build
+  OPENVINO_REPO: /__w/openvino/openvino/openvino
+  INSTALL_TEST_DIR: /__w/openvino/openvino/openvino_install/tests/
+  INSTALL_DIR: /__w/openvino/openvino/openvino_install
```

Comment:
Since every job runs from scratch and has its own environment, I would suggest keeping the path-like variables in the per-job `env`. The workflow level `env` could be left with stuff like `TBB_ENABLE_SANITIZERS` and such.
The `/__w/openvino/openvino/openvino` prefix for paths could be different for jobs if, for example, we clone in a different directory or if we change the workspace on the runners. 
You could use `${{ github.workspace }}` like in https://github.com/openvinotoolkit/openvino/blob/39418cadb9d53cd4dc45bc49c107220a45e1122c/.github/workflows/manylinux_2014.yml#L99 if you add the mount for container like in: https://github.com/openvinotoolkit/openvino/blob/39418cadb9d53cd4dc45bc49c107220a45e1122c/.github/workflows/manylinux_2014.yml#L95

Reply:
Fine. I'll apply it in refactoring PR (162531)